ServiceWebsite/ServiceWebsite.AcceptanceTests/Features/ThankYou.feature

```diff
@@ -14,3 +14,34 @@ Scenario: Representative participant has camera and microphone on laptop or desk
 	Then Representative should be on 'questionnaire completed' screen
 	When proceeds to next page
 	Then Representative should be on 'check your computer' screen
+
+@smoketest @VIH-4578
+Scenario: Representative views thank you page after submitting self test questionnaire completed page for a client who is suitable for a video hearing
+	Given Representative participant starts the questionnaire
+	When provides answer
+	 | page                      | answer  |
+	 | about you                 | No      |
+	 | access to a suitable room | Yes     |
+	 | about your client         | No     |
+	 | client attendance         | No      |
+	 | hearing suitability       | No      |
+	 | your computer             | Yes     |
+	 | about your computer       | Yes |
+
+	Then Representative should be on 'questionnaire completed' screen
+	And all the answers should match
+	And a link with text 'Print my answers' to print the answers should be visible
```

Comment:
For example, I don't think we need to test this in this test as it's covered elsewhere

Reply:
Removed the step